packs/email/sensors/imap_sensor.py

```diff
@@ -41,17 +41,18 @@ def __init__(self, sensor_service, config=None, poll_interval=30):
     def setup(self):
         self._logger.debug('[IMAPSensor]: entering setup')
 
-        if 'imap_mailboxes' in self._config:
-            self._parse_mailboxes(self._config['imap_mailboxes'])
-
     def poll(self):
         self._logger.debug('[IMAPSensor]: entering poll')
 
+        if 'imap_mailboxes' in self._config:
+            self._parse_mailboxes(self._config['imap_mailboxes'])
```

Comment:
Just want to confirm - we re-establish a connection to the IMAP server on every poll now, right?